### ISSUE TITLE
Hyper-parameters for ResNet-50

### ISSUE THREAD

**Issue by zhaozhengChen** [liuweide01/RPNet-Weakly-Supervised-Segmentation#2]
Hi authors,

Do you use the same parameters for ResNet-50?
When I run the code with ResNet-50, I got the following results:

{'iou': array([0.71786668, 0.32337534, 0.22839967, 0.37651976, 0.28739013,                                                                                  
       0.45902408, 0.62007131, 0.48607532, 0.60152633, 0.28056315,                                                                                          
       0.50565703, 0.53805612, 0.53191146, 0.53455538, 0.54745629,                                                                                          
       0.49557657, 0.41466069, 0.51863075, 0.45559653, 0.46711257,                                                                                          
       0.44537695]), 'miou': 0.46835248169777155}

It is even worse than baseline (0.483).

**Comment by liuweide01** [comment]
You can reduce the learning rate.
